spec/integration/init_web_spec.cr

```diff
@@ -45,22 +45,31 @@ describe "Initializing a new web project" do
 
   it "creates an api only web app successfully" do
     puts "Api only: Running integration spec. This might take awhile...".colorize(:yellow)
-    should_run_successfully "crystal run src/lucky.cr -- init.custom test-project test-project --api"
+    should_run_successfully "crystal run src/lucky.cr -- init.custom test-project --api"
     compile_and_run_specs_on_test_project
   end
 
   it "creates an api only app without auth" do
     puts "Api only without auth: Running integration spec. This might take awhile...".colorize(:yellow)
-    should_run_successfully "crystal run src/lucky.cr -- init.custom test-project test-project --api --no-auth"
+    should_run_successfully "crystal run src/lucky.cr -- init.custom test-project --api --no-auth"
     compile_and_run_specs_on_test_project
   end
 
   it "creates a full app without auth" do
     puts "Web app without auth: Running integration spec. This might take awhile...".colorize(:yellow)
-    should_run_successfully "crystal run src/lucky.cr -- init.custom test-project test-project --no-auth"
+    should_run_successfully "crystal run src/lucky.cr -- init.custom test-project --no-auth"
     compile_and_run_specs_on_test_project
   end
 
+  it "creates a full app in a different directory" do
+    puts "Web app with custom directory: Running integration spec. This might take awhile...".colorize(:yellow)
```

Comment:
You can probably remove the part about it taking awhile. I think this should be a second or two at most